libraries/google_bigquery_dataset.rb

```diff
@@ -70,4 +63,14 @@ def parse_time_string(time_string)
   def exists?
     !@fetched.nil?
   end
+
+  private
+
+  def _base
```

Comment:
From what I've seen, Ruby doesn't seem to do the `_method` notation to say something is private. They either put the method under the private section or (less so) write `private def method`. Python definitely does the `_method` thing for private methods and I feel like that's where it's most famously done.

Some people do place an underscore and honestly, I don't really care either way. Your call.

Reply:
Yeah, I was searching around for some kind of Ruby prefix for indicating that it is private. Seems like some people use `_` but it's not standard. I needed to prefix these methods so they won't conflict with resource properties, or else rename them to something more specific.